src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -122,16 +126,27 @@ for further details.
 	margin-bottom: 32px;
 }
 
-.sidecarWrapper {
+.sidecarPosition {
+	--breadcrumb-bar-height: 58px;
+	--sidecar-position-top: calc(
+		var(--sticky-bars-height) + var(--main-area-padding-top) +
+			var(--breadcrumb-bar-height)
+	);
```

Comment:
@ashleemboyer Great call 💖 I [considered it](https://github.com/hashicorp/dev-portal/pull/1661/commits/e2dad01ecde1e8c505d244b9d72576d12032f7c0#diff-0c99b3fcf37ed6bb4819102c69efa12cc6c55e69153cd52fede833b259810dc1R141) but was unsure about the tradeoff of trying to dive in and refactor the layout a bit, since the downside here would be a bit of misalignment rather than something totally broken, and it seemed _possible_ the refactor could spiral into deeper work.

That being said, deadline here isn't tight, so I took another look and have just pushed up some refactoring to `sidebar-sidecar.module.css`. Here's a quick summary, `before` on the left, `after` on the right:

<img width="2560" alt="before-after" src="https://user-images.githubusercontent.com/4624598/221188740-4c556c33-5e6b-406c-82e5-4a27b438b02a.png">

I think this should work as expected, and continue to match designs, while removing the brittleness of relying on a specific pixel value for the breadcrumb bar height. This should also help prevent breadcrumb wrapping in some cases:

| Before | After |
| - | - |
| ![before](https://user-images.githubusercontent.com/4624598/221191226-38388955-8926-41d1-8f99-1fe77d3ae37e.png) | ![after](https://user-images.githubusercontent.com/4624598/221191231-49ebfa3d-fe7c-4c43-a5c0-ac0512222297.png) |

Let me know what you think 🙇